security-analysis/security-analysis-api/src/test/java/com/powsybl/security/json/JsonActionAndOperatorStrategyTest.java

```diff
@@ -47,8 +47,8 @@ void actionRoundTrip() throws IOException {
         List<Action> actions = new ArrayList<>();
         actions.add(new SwitchAction("id1", "switchId1", true));
         actions.add(new MultipleActionsAction("id2", Collections.singletonList(new SwitchAction("id3", "switchId2", true))));
-        actions.add(new LineConnectionAction("id3", "lineId3", true, true));
-        actions.add(new LineConnectionAction("id4", "lineId4", false));
+        actions.add(new TerminalConnectionAction("id3", "lineId3", true)); // both sides.
+        actions.add(new TerminalConnectionAction("id4", "lineId4", false));
```

Comment:
might want to test also with specifying side

Reply:
Yes of course, it was just for transparency that I have open the PR, because I have started (and stopped) the OLF integration.